libs/ledger-live-common/src/hw/signMessage/index.test.ts

```diff
@@ -9,7 +9,7 @@ describe("prepareMessageToSign", () => {
     // Given
     const crypto = createCryptoCurrency("ethereum");
     const account = createAccount(crypto);
-    const message = "Message de test";
+    const message = "4d6573736167652064652074657374";
```

Comment:
Does it makes sense to keep the message to use it in the `expect` line 30 and use`Buffer` to create the `hex` version of the message ?
```suggestion
    const message = "Message de test";
    const hexMessage = Buffer.from(message).toString("hex");
```
or
```suggestion
    const clearMessage = "Message de test";
    const message = Buffer.from(clearMessage).toString("hex");
```

Reply:
I am not sure to understand. This is the expected behavior.
If you ask me about the format (with the `message` property and the `rawMessage` property), it is a format already defined in ledger-common and ledgerjs.

Reply:
This is just an improvement for the maintainability of the test so that we only need to change one string and not 2 or more.

Reply:
Ok, I think I understand your point.
I don't want to convert the human readable message with Buffer inside the test, because I want the test to be dumb and not carrying some logic.